addons/mail/wizard/mail_compose_message.py

```diff
@@ -559,7 +559,7 @@ def _get_optout_emails(self, mail_values_dict):
         return []
 
     def _onchange_template_id(self, template_id, composition_mode, model, res_id):
-        """ - mass_mailing: we cannot render, so return the template values
+        r""" - mass_mailing: we cannot render, so return the template values
```

Comment:
Isn't it weird to mark docstring as regex string just for `/!\` to not raise the linter error? Wouldn't a `# noqa:` notation be more logical?

Reply:
It not a linting issue but a parsing issue, in python 3.14 this code won't even run.

Reply:
Why don't you remove /!\ then ?